Added Action settings: HasRetry, RetryCount and RetryWait

brzezin87 opened the pull request:
Added Action settings:
- HasRetry
- RetryCount
- RetryWait

Added new BESCoreElement function:
- _create_child_elem_with_attributes

Commit:
Added Action settings: HasRetry, RetryCount and RetryWait

Spajderix reviewed (changes requested):
- modify _create_child_elem so that at the end it returns new node, then modify _create_child_elem_with_attributes so that it uses the former and just adds additional attribute
- since _create_child_elem_with_attributes is in plural, then it should use dict for a list of attributes and their values so that more than one can be added
- RetryCount is defined in BES.xsd as type="xs:nonNegativeInteger", please also verify non-negativity
- Add a property RetryWaitBehavior, so that "Behavior" attribute of RetryWait node can also be easily modified. Validate value of the attribute as it's set.

Commit:
- non-negativity for RetryCound added
- RetrywaitBehavior property added
- _create_child_elem_with_attributes now uses _create_child_elem function
- _create_child_elem_with_multiple_attributes function added for future functionality

Commit:
- code cleaning
